src/util.rs

```diff
@@ -392,8 +397,11 @@ pub fn markdown_to_kakoune_markup<S: AsRef<str>>(markdown: S) -> String {
                 }
                 Tag::BlockQuote => {
                     has_blockquote_text = false;
-                    is_blockquote = false
+                    is_blockquote = false;
+                    let base_face = pop_base_face(&mut face_stack, FACE_DEFAULT);
+                    markup.push_str(&format!("{{{}}}", base_face));
                 }
+                // Don't refactor this into the next arm. There is an additional '\n' here.
```

Comment:
I'm not a big fan of comments that only reiterate what the code is doing.
```rust
// add the suffix
line += suffix;
```

The primary goal of this comment is to prevent someone starting to merge this into the next arm, only to realize halfway through that there is a difference of a single character.
I consider the callout `Don't refactor this` more obvious in that regard than comparing the existence of a comment somewhere in the arm's code block, especially when just skimming the code.
And once someone does know that there is a difference and wants to find it, spotting that specific character is relatively easy.

Reply:
This will be obsolete since I'm sorting the cases in declaration order